### PR TITLE
Chair review

### PR DESCRIPTION
**Pull request by cabo**

### CODE REVIEW

File: draft-ietf-core-groupcomm-bis.md [hunk 1]
```diff
@@ -234,9 +229,9 @@ Beyond this particular case, applications should be careful in associating a sam
 
 {{fig-group-relation-example}} provides a deployment example of the relations between the different types of groups. It shows six CoAP servers (Srv1-Srv6) and their respective resources hosted (/resX). There are three application groups (1, 2, 3) and two security groups (1, 2). Security Group 1 is used by both Application Group 1 and 2. Three clients (Cli1, Cli2, Cli3) are configured with security material for Security Group 1. Two clients (Cli2, Cli4) are  configured with security material for Security Group 2. All the shown application groups use the same CoAP group (not shown in the figure), i.e., one specific multicast IP address and UDP port number on which all the shown resources are hosted for each server.
 
-~~~~~~~~~~~
- ________________________________    _________________________________
-/                                \  /                                 \
+~~~~~~~~~~~ aasvg
```
Comment (EskoDijk) [inline draft-ietf-core-groupcomm-bis.md:232]: My kramdown-rfc2629 1.6.6 gives an error on this "No such file or directory - aasvg"

**Review by EskoDijk — APPROVED** [review]
Agree with all the improvements, thanks for this Carsten!
One issue I had with the compilation for the "aasvg" option on two figures - on my Windows MSYS2 setup the most recent kramdown-rfc2629 version seems to be 1.6.6.

**Comment by EskoDijk** [comment]
@cabo Asking this for another I-D: is putting the names of contributors in brackets like "{{{Christian Amsüss}}}, {{{Carsten Bormann}}}, " etc. sufficient to make these persons as contributors to the document, and so that IETF can keep statistics on this?

I heard from another source that the names need to be in a "Contributors" section for the IETF to count this and not in an "Acknowledgements" section. But that may be wrong.

**Review by marco-tiloca-sics — APPROVED** [review]
Thanks for reviewing and for the proposed changes!

**Pull request merged**